posthog/api/decide.py

```diff
@@ -102,6 +102,23 @@ def get_decide(request: HttpRequest):
                 generate_exception_response("decide", f"Malformed request data: {error}", code="malformed_data"),
             )
 
+        if not "distinct_id" in data:
+            capture_exception(
```

Comment:
Why not raise validation error here? You would get the same result without repeating yourself I think.

Reply:
The reason that wouldn't work is because this view is not on DRF, therefore the error handler doesn't apply, so if we raise a `ValidationError`, the response will look different from our standardized error responses.